deps/SnoopCompile/snoop_bot_config.jl

```diff
@@ -0,0 +1,7 @@
+using CompileBot
+
+botconfig = BotConfig(
+    "Plots",
+    version = [v"1.6"],
+    else_version = v"1.7",
```

Comment:
I think I am in favor of putting `v"1.7"` also in `version`. IIUC then you might get a slower version of Plots on other versions, but can't get a broken version, or am I wrong?

Reply:
It looks like a safer approach.